girder/api/v1/item.py

```diff
@@ -276,46 +249,35 @@ def deleteItem(self, item, params):
         return {'message': 'Deleted item %s.' % item['name']}
 
     @access.public(scope=TokenScope.DATA_READ)
-    @loadmodel(model='item', level=AccessType.READ)
-    @describeRoute(
+    @autoDescribeRoute(
         Description('Get the path to the root of the item\'s hierarchy.')
-        .param('id', 'The ID of the item.', paramType='path')
+        .modelParam('id', model='item', level=AccessType.READ)
         .errorResponse('ID was invalid.')
         .errorResponse('Read access was denied for the item.', 403)
     )
     def rootpath(self, item, params):
         return self.model('item').parentsToRoot(item, self.getCurrentUser())
 
     @access.user(scope=TokenScope.DATA_WRITE)
-    @loadmodel(model='item', level=AccessType.READ)
     @filtermodel(model='item')
-    @describeRoute(
+    @autoDescribeRoute(
         Description('Copy an item.')
```

Comment:
'Copy an item, into the parent folder of the source item if no folderId is specified.'